README.md

```diff
@@ -16,31 +16,24 @@ The SecretHub CLI provides the command-line interface to interact with SecretHub
 
 > [SecretHub][secrethub] is a developer tool to help you keep database passwords, API tokens, and other secrets out of IT automation scripts.
 
-## Installation
-
-### Download a binary distribution
+## Usage
 
-Official binary distributions are available for the Linux, macOS, and Windows operating systems for both the 32-bit (386) and 64-bit (amd64) versions.
-You can find the latest release [here][releases].
+```sh
+$ secrethub write path/to/secret
+Type in your secret: ************************  
 
-To install the SecretHub CLI, download the archive file appropriate for your operating system and extract it e.g. to `/usr/local/secrethub`.
+$ cat config.yml.tpl
+db_user: myapp
+db_password: ${path/to/secret:latest}
 
-```sh
-mkdir -p /usr/local/secrethub
-tar -C /usr/local/secrethub -xzf secrethub-VERSION-OS-ARCH.tar.gz
+$ cat config.yml.tpl | secrethub inject
+db_user: myapp
+db_password: LEYkTdMCksCVMc4X3gpYN0fk
 ```
```

Comment:
Add a line under the code example saying something like: `See the [getting started guide](...) for a more extensive introduction and check out the [CLI reference docs](...) for an overview of all commands.`